src/utils/ERC20CosmosCoinAmountConversion.sol

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.25;
+
+// https://docs.openzeppelin.com/contracts/4.x/api/token/erc20#IERC20Metadata
+import { IERC20Metadata } from "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+
+library ERC20CosmosCoinAmountConversion {
```

Comment:
I think we should refer to the coins as `sdk.Coin`? We could also expand the use case of this library in the future to include other `sdk.Coin` utility functions if needed So the name of the lib would be:
```suggestion
library SdkCoin {
```
What do you think?

Reply:
Make sense!